Directory.Packages.props

```diff
@@ -0,0 +1,38 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Lombiq.Analyzers.OrchardCore" Version="5.0.0" />
+    <PackageVersion Include="Lombiq.HelpfulLibraries.OrchardCore" Version="9.1.0" />
+    <PackageVersion Include="Lombiq.NodeJs.Extensions" Version="2.1.0" />
+    <PackageVersion Include="Lombiq.Tests" Version="3.0.0" />
+    <PackageVersion Include="Lombiq.Tests.UI" Version="9.0.1-alpha.2.occ-218" />
+    <PackageVersion Include="Lombiq.Tests.UI.AppExtensions" Version="9.0.1-alpha.2.occ-218" />
+    <PackageVersion Include="Lombiq.Tests.UI.Shortcuts" Version="9.0.1-alpha.2.occ-218" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="OrchardCore.Application.Cms.Targets" Version="1.8.2" />
```

Comment:
Is the OC 1.8.2 upgrade here deliberate?

Reply:
Yes, the substantive fix was in UITT, which uses OC 1.8.2. So if we want to use the new alpha of UITT, we have to upgrade to OC 1.8.2. That's why the PR targets `issue/OCC-223` instead of `dev`.